raise Exception('Testing raise Exception("...")')
